bullet_train-obfuscates_id/app/models/concerns/obfuscates_id/sluggable.rb

```diff
@@ -0,0 +1,52 @@
+module ObfuscatesId
+  # We only want to ensure slugging is possible if a developer
+  # can generate a `slug` field partial with our scaffolder.
+  if defined?(BulletTrain::SuperScaffolding)
```

Comment:
Considering changing this, it may not be too big of a deal if `BulletTrain::SuperScaffolding` is defined or not (maybe I can just write a comment saying the the module is only intended for use with `BulletTrain::SuperScaffolding`).